test_crates/trait_associated_const_added/new/src/lib.rs

```diff
@@ -0,0 +1,48 @@
+mod sealed {
+    pub(crate) trait Sealed {}
+}
+
+// trigger
+pub trait WillGainConstWithoutDefault {
+    const BAR: bool;
+
+    fn make_me_object_safe() -> Self;
+}
+
+pub trait WillGainConstWithoutDefaultSealed: sealed::Sealed {
+    const BAR: bool;
+
+    fn make_me_object_safe() -> Self;
+}
+// trigger
+pub trait WillGainConstWithoutDefaultAndSeal: sealed::Sealed {
+    const BAR: bool;
+
+    fn make_me_object_safe() -> Self;
+}
+
+pub trait WillGainConstWithDefault {
+    const BAR: bool = true;
+
+    fn make_me_object_safe() -> Self;
+}
+pub trait WillGainConstWithDefaultSealed: sealed::Sealed {
+    const BAR: bool = true;
+
+    fn make_me_object_safe() -> Self;
+}
+
+// trigger
+pub trait WillGainAnotherConstWithoutDefault {
+    const ONE: bool;
+    const TWO: bool;
+
+    fn make_me_object_safe() -> Self;
+}
+// trigger
+pub trait WillGainAnotherConstWithoutDefaultSealed: sealed::Sealed {
```

Comment:
I think this shouldn't trigger, since in `old` the trait is sealed. Perhaps you didn't intend to seal it in the baseline?

Reply:
no you are right, it should not indeed trigger